service/src/main/java/com/github/swapbook/repositories/UsersRepository.java

```diff
@@ -0,0 +1,30 @@
+package com.github.swapbook.repositories;
+
+import com.github.swapbook.model.User;
+import org.springframework.stereotype.Component;
+
+import java.util.LinkedList;
+import java.util.List;
+
+
+@Component
+public class UsersRepository {
```

Comment:
Tutaj potrzeby jest interfejs bo jak dodamy baze danych to tylko zmienisz w klasach. ( w C# np jest tak:
IUserRepository userRepository = new LocalUserRepository(); //z lokalna listą tak jak teraz masz lub często nazywane FakeRepository
albo  
IUserRepository userRepository = new DBUserRepository(); //z całym łączeniem z bazą danych
Myślę że ta konwencja będzie lepsza.

Reply:
zrobione